src/resources/trnmanager.c

```diff
@@ -9,6 +9,12 @@
 #include <stdio.h>
 #include <string.h>
 
+static int trn_sort_compare_fn(const void *a, const void *b) {
+    sd_tournament_file const *trn_a = a;
+    sd_tournament_file const *trn_b = b;
+    return (trn_a->registration_fee > trn_b->registration_fee) - (trn_a->registration_fee < trn_b->registration_fee);
```

Comment:
What's the problem with just `return trn_a->registration_fee - trn_b->registration_fee;` here?

Reply:
i don't want to overflow or underflow an integer.